sklearn/ensemble/gradient_boosting.py

```diff
@@ -850,25 +850,26 @@ def _check_params(self):
             if self.max_features == "auto":
                 # if is_classification
                 if self.n_classes_ > 1:
-                    max_features = max(1, int(np.sqrt(self.n_features)))
+                    max_features = max(1, int(np.sqrt(self.n_features_)))
```

Comment:
If this is to be a public attribute, it should be documented in all applicable classes' docstrings.